department-of-veterans-affairs/va.gov-cms
# VACMS-2332

## Description
See #2332 

## Testing done
Visual

## QA steps
As an admin, go to `admin/config/system/taxonomy-entity-index/reindex` and rebuild content index.
Go to `/section/veterans-health-administration/visn-4` and visually verify that content associated with visn-4 (and children) appears
